chapters/copd/main.tex

```diff
@@ -837,65 +868,109 @@ \subsection{Moving arrivals between clusters}\label{subsec:moving}
-resources can be reduced by moving arrivals from Cluster 0 to either Cluster 1
-or Cluster 2, i.e.\ by reducing the overall resource requirements of such spells.
-
-In essence, this entire analysis offers two messages: that there are several
-ways in which the system can get worse and even overwhelmed but, more
-importantly, that any meaningful impact on the system must come from a stimulus
-outside of the system that results in more healthy patients arriving at the
-hospital. This conclusion is non-trivial; the first two scenarios in this
-analysis show that there are no quick solutions to reduce the effect of COPD
-patients on hospital capacity or length of stay. The only effective intervention
-is found through inter-cluster transfers.
-
 \begin{figure}
     \centering
-    \includegraphics[width=\imgwidth]{moving_time}
+    \includegraphics[width=\textwidth]{moving_time}
     \caption{%
         Plots of proportions of each cluster moving to another against relative
-        system time.
+        system time
     }\label{fig:moving_time}
 \end{figure}
 
 \begin{figure}
     \centering
-    \includegraphics[width=\imgwidth]{moving_util}
+    \includegraphics[width=\textwidth]{moving_util}
     \caption{%
         Plots of proportions of each cluster moving to another on relative
-        server utilisation.
+        server utilisation
     }\label{fig:moving_util}
 \end{figure}
 
-\section{Conclusion}\label{sec:conclusion}
+Considering Figure~\ref{fig:moving_time}, it appears that each type of transfer
+falls into one of two categories: either completely derailing the system (such
+as moving any cluster to Cluster 3) or improving system times, albeit mildly. 
+The latter case occurs in the following transfers:
+
+\begin{itemize}
+    \item Cluster 0 to Clusters 1 or 2
+    \item Cluster 1 to Cluster 2
+    \item Cluster 3 to any other cluster
+\end{itemize}
+
+A finer look at the effect of these transfer types on relative system times is
+given in Table~\ref{tab:moving_time}. Likewise, their effects on relative server
+utilisation is given in Table~\ref{tab:moving_util}. 
+
+\begin{table}
+    \centering%
+    \resizebox{\textwidth}{!}{%
+        \input{chapters/copd/paper/tex/moving_time.tex}
+    }
+    \caption{%
+        Proportional changes in median relative system time for selected cluster
+        transfers
+    }\label{tab:moving_time}
+\end{table}
+
+\begin{table}
+    \centering%
+    \resizebox{\textwidth}{!}{%
+        \input{chapters/copd/paper/tex/moving_util.tex}
+    }
+    \caption{%
+        Proportional changes in median relative utilisation for selected cluster
+        transfers
+    }\label{tab:moving_util}
+\end{table}
+
+The message delivered by these transfers is that in order to improve system
+times in hospitals, the patients arriving at hospital must present with fewer
+resource requirements. Meanwhile, the complexity of their condition is less
```

Comment:
This first sentence is a very good point **but** it's also somewhat obvious. I think a wording that tries to emphasise that this is the **only** way to reduce times is better.

Reply:
Fixed by 02594a8